js/util-galery.js

```diff
@@ -0,0 +1,68 @@
+
+const bigPhoto = document.querySelector('.big-picture');
+const body = document.querySelector('.body');
+const commentCounter = document.querySelector('.social__comment-count');
+const commentsLoader = document.querySelector('.comments-loader');
+const closeButton = document.querySelector('.big-picture__cancel  cancel');
+const socialComments = document.querySelector('.social__comments');
+
+const hideBigPhoto = () => {
+  bigPhoto.classList.add('hidden');
+  body.classList.remove('modal-open');
+  commentCounter.classList.remove('hidden');
+  commentsLoader.classList.remove('hidden');
+  //document.removeEventListener('keydown', onDocumentKeydown);
+  //closeButton.removeEventListener('click', onCloseButton);
+};
+
+const onDocumentKeydown = (evt) => {
+  if (evt.key === 'Escape') {
+    evt.preventDefault();
+    hideBigPhoto();
+  }
+};
+
+const onCloseButton = () => {
+  hideBigPhoto();
+};
+
+const renderPhotoDetails = ({url, description, likes, comments}) => {
+  bigPhoto.querySelector('.big-picture__img').src = url;
```

Comment:
Селектор немного не тот `.big-picture__img` это блок div, a нам нужно изображение внутри блока, т.е. тут селектор будет вот таким:
```suggestion
  bigPhoto.querySelector('.big-picture__img img').src = url;
```
Грубо говоря конструкция выше это тоже самое что мы сначала ищем блок с изображением `.big-picture__img`, а потом в нем ищем сам элемент изображения по тегу `img`
```suggestion
  bigPhoto.querySelector('.big-picture__img').querySelector('img').src = url;
```